wordpress-mobile/gutenberg-mobile
# Correct slider step value

Fixes #2117 
Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/21451

To test:

Open mobile app
Add spacer block
Open spacer settings
Move slider
Expect that step value is 1

PR submission checklist:
